src/Api/SyncApiClient.php

```diff
@@ -36,21 +43,24 @@ public function __construct($psAccounts, $syncApiUrl, $module)
     {
         $this->module = $module;
         $this->jwt = $psAccounts->getPsAccountsService()->getOrRefreshToken();
+        $this->shopId = $psAccounts->getPsAccountsService()->getShopUuid();
         $this->syncApiUrl = $syncApiUrl;
     }
 
     /**
      * @see https://docs.guzzlephp.org/en/stable/quickstart.html-
      *
+     * @param int $timeout
+     *
      * @return HttpClientInterface
      */
-    private function getClient()
+    private function getClient($timeout = Config::SYNC_API_MAX_TIMEOUT)
```

Comment:
this `SYNC_API_MAX_TIMEOUT` variable seems not to be defined anywhere, am I right?

Reply:
```
class Config
{
    public const SYNC_API_MAX_TIMEOUT = 5;

```
??? :-)